queries/cylc/injections.scm

```diff
@@ -0,0 +1,22 @@
+((comment) @injection.content
+  (#set! injection.language "comment"))
+
+((setting
+  key: (key) @key
+  (#match? @key "^script$|-script$|^script-")
+  value: (_
+    (string_content) @injection.content))
+  (#set! injection.language "bash"))
+
+; Requires no spacing around "=" in environment settings for proper highlighting.
```

Comment:
It seems like maybe this whole thing should be commented out then (or just removed), I have seen a lot of files with spaces between `=` (which is not valid bash) e.g. https://github.com/hjoliver/bomdr/blob/99354074846f738c6c13d8e747db7a7696855089/model1/flow.cylc#L5

Also there can be jinja expressions in here looks like, which is also not valid bash and should be handled more specifically

Reply:
Okay. I'll reimplement it if `tree-sitter` add support for the #3625 proposal.

Reply:
Wrong link?